noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/TwitterFeedStreamDriver.java

```diff
@@ -21,11 +22,11 @@
 
     Client twitterClient;
     volatile boolean isConnected = false;
-    FeedSocketAdapterClient socketAdapterClient;
+    public FeedSocketAdapterClient socketAdapterClient;
```

Comment:
I changed it so I could reuse the function "openSocket" and also the socketAdapterClient it creates. Therefore, I think a better approach would be moving this function from TwitterFeedStreamDriver to FeedSocketAdapterClient so I could use it in both places.